app/routes/send-endringsmelding.module.css

```diff
@@ -1,10 +1,16 @@
 .navigeringsKnapper {
+  margin-top: 5rem;
   min-width: 17rem;
   display: flex;
   flex-direction: row;
-  justify-content: space-evenly;
+  justify-content: center;
+  gap: 1rem;
 }
 
 .fullBredde {
   width: 100%;
 }
+
+.toppMargin {
+  margin-top: 5rem;
+}
```

Comment:
Foretrekker bruk av flexbox men skal la denne gå for å ta styling av alert når vi har bestemt hvordan den skal se ut